packages/lit-dev-content/site/docs/templates/expressions.md

```diff
@@ -137,14 +137,21 @@ html`<main>${bodyText}</main>`
 
 Expressions in the child position can take many kinds of values:
 
-* Primitive values likes strings, numbers, and booleans
+* Primitive rendering values
+* Non-rendering values
 * `TemplateResult` objects created with the `html` function.
 * DOM nodes
 * Arrays or iterables of any of the supported types
 
-### Primitive values
+### Primitive rendering values
 
-Primitives values like strings, numbers, booleans, null, and undefined are converted to strings when interpolated into text content or attribute values. They are checked for equality with the previous value so the DOM is not updated if the value hasn't changed.
+Strings, numbers, and booleans are all converted to strings when interpolated into text content. This means that a value of `5` will render the string `'5'`, but likewise a value of `true` will render `'true'` and a value of `false` will render `'false'`. Rendering a boolean like this is uncommon, and these values are typically rendered using conditionals. For more on conditionals, see [Conditionals](/docs/templates/conditionals/).
```

Comment:
> Rendering a boolean like this is uncommon, and these values are typically rendered using conditionals.

Don't totally follow this. What is "these values" referring to, booleans? You don't _render a boolean_ using a conditional. Do you mean to say something like "booleans are typically used when rendering conditionals or in boolean attribute expressions"?

Reply:
> They are checked for equality with the previous value so the DOM is not updated if the value hasn't changed.

Was this part intentionally deleted?

Reply:
Yes, it was the only place where the update behavior was addressed. It's an odd place to put that imo.